src/main/resources/travis/travis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
In theory the enc file could go in private centaur, depends on how much ppl care right now